health_ping.py

```diff
@@ -22,6 +22,19 @@
 refresh_interval = int(os.getenv("REFRESH_INTERVAL","60"))
 log_level = os.environ.get('LOG_LEVEL', 'INFO').upper()
 
+# limit results for testing/dev
+# See strapi filter syntax https://docs.strapi.io/dev-docs/api/rest/filters-locale-publication
+# Example filter string = '&filters[name][$contains]=example'
+# sc_filter = '&filters[name][$contains]=book-a-prison-visit-staff-ui'
+sc_filter = os.getenv("sc_filter")
+sc_page_size = 10
+sc_pagination_page_size = f"&pagination[pageSize]={sc_page_size}"
+# Example Sort filter
+#SC_SORT='&sort=updatedAt:asc'
+sc_sort = ''
+sc_endpoint = f"{sc_api_endpoint}/v1/components?populate=environments{sc_filter}{sc_pagination_page_size}{sc_sort}"
```

Comment:
this is duplicated [line](https://github.com/ministryofjustice/hmpps-health-ping/pull/10/files#diff-73b875e568a6d799661f8bfb64f865de2790269e12df6410363ce2e9ed847a37L213)